pkg/sql/sqlbase/structured.proto

```diff
@@ -129,6 +129,19 @@ message ColumnDescriptor {
   // Expression to use to compute the value of this column if this is a
   // computed column.
   optional string compute_expr = 11;
+  // LogicalColumnID is lazily set, do not access directly,
```

Comment:
[nit]: Say something like "LogicalColumnID must be accessed through the accessor ... it is incorrect to access it directly.